lib/actions/push.ts

```diff
@@ -243,6 +244,14 @@ export const push: CommandDefinition<
 				left hand side of the = character will be treated as the variable name.
 			`,
 		},
+		{
+			signature: 'convert-eol',
+			alias: 'l',
+			description: stripIndent`
+				During push, convert line endings from CRLF (Windows format) to LF (Unix format) in text based files.
+				Note that this does not alter the source files, but converts them on-the-fly during push.`,
```

Comment:
Perhaps what you meant by "During push" is "on the fly", _"over the network stream that the CLI sends to balenaCloud, docker daemon or balenaEngine on a balenaOS device",_ which is why source files are not modified. Perhaps what users would find easier to understand is "in memory", i.e. not persistent -- or is this an alien concept to [Generation Z](https://en.wikipedia.org/wiki/Generation_Z)? :-)  So, another suggestion:
```
	Perform in-memory conversion of line endings from CRLF (Windows format)
	to LF (Unix format) in text based files. Changes are not saved to disk.
```
Btw, is saying "text based files" really clearer than "text files"? I take it to mean "not just `"*.txt"` files but also source code like shell scripts", but I'd have hoped that users would get that... Just my 2 cents.

Reply:
> in-memory

I've always understood "in-memory" to mean that the data to be operated on is loaded into memory in its entirety for processing (a la DB anti-patterns), and for me it doesn't say anything about whether the results are persisted or not.   This stuff is quite subjective I suppose.

> Changes are not saved to disk.

For me this is potentially ambiguous, since the changes *are* saved to disk on the device.  IMO "does not alter the source files" is more explicit.

> Btw, is saying "text based files" really clearer than "text files"? I take it to mean "not just "*.txt" files but also source code like shell scripts", 

To me it is, but again I guess it's very subjective.  If I heard someone say "a text file" I would not immediately think of shell scripts, json, js etc., even though they are of course technically text files.  I think the problem comes from the fact that the term is used to describe concepts at multiple levels of abstraction. How about "non-binary"? :laughing:

Reply:
How about simply:

> Convert line endings from CRLF (Windows format) to LF (Unix format).  Source files are not modified.

I think we can drop the whole "text files" bit, since I guess it's obvious that binary files don't have lines or line endings :)

Reply:
That's good. I was overthinking it. :-)